src/collective/cover/browser/cover.py

```diff
@@ -114,11 +99,8 @@ def render(self):
             return 'Widget does not exist'
 
 
-class RemoveTileWidget(grok.View):
+class RemoveTileWidget(DefaultView):
```

Comment:
I was wondering the same, but that's what we did in collective.nitf; please do some research about it.